staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -569,6 +569,25 @@ func isExpiredError(err error) bool {
 	return apierrors.IsResourceExpired(err) || apierrors.IsGone(err)
 }
 
+var tooLargeResourceVersionCauseMsg = "Too large resource version"
+
 func isTooLargeResourceVersionError(err error) bool {
-	return apierrors.HasStatusCause(err, metav1.CauseTypeResourceVersionTooLarge)
+	if apierrors.HasStatusCause(err, metav1.CauseTypeResourceVersionTooLarge) {
+		return true
+	}
+	// In Kubernetes 1.18.5 and earlier, the api server doesn't set status cause metav1.CauseTypeResourceVersionTooLarge
```

Comment:
nit: can you please split those comment lines to make them a bit shorter?

Reply:
done